proposals/lip-0053.md

```diff
@@ -115,7 +115,7 @@ The certificate schema `certificateSchema` is defined in [LIP 0061][lip-0061#cer
 #### verifyLivenessConditionForRegisteredChains
 
 ```python
-def verifyLivenessConditionForRegisteredChains(ccu: Transaction) -> None:
+def verifyLivenessConditionForRegisteredChains(ccu: CCU) -> None:
      certificate = decode(certificateSchema, ccu.params.certificate)
```

Comment:
In principle we should validate the schema here, as this is the first time we decode the certificate bytes

Reply:
I would agree that we should validate the schema as early as possible, see also discussion above.